test/proto/nested/nested_pb2.pyi.expected

```diff
@@ -54,9 +54,12 @@ class AnotherNested(google___protobuf___message___Message):
         def values(cls) -> typing___List[AnotherNested.NestedEnum]: ...
         @classmethod
         def items(cls) -> typing___List[typing___Tuple[str, AnotherNested.NestedEnum]]: ...
-    INVALID = typing___cast(NestedEnum, 0)
-    ONE = typing___cast(NestedEnum, 1)
-    TWO = typing___cast(NestedEnum, 2)
+        INVALID = typing___cast(AnotherNested.NestedEnum, 0)
+        ONE = typing___cast(AnotherNested.NestedEnum, 1)
+        TWO = typing___cast(AnotherNested.NestedEnum, 2)
+    INVALID = typing___cast(AnotherNested.NestedEnum, 0)
+    ONE = typing___cast(AnotherNested.NestedEnum, 1)
+    TWO = typing___cast(AnotherNested.NestedEnum, 2)
```

Comment:
Would you mind adding a line to `test_func` in test/test_generated_mypy.py to cover this new usage?

Reply:
Sure